src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -1,14 +1,19 @@
 # -*- coding: utf-8 -*-
+import logging
 from enum import Enum
 from typing import NamedTuple
 
 from .. import mgr
 
+logger = logging.getLogger('rgw_client')
```

Comment:
Why is this logger named `rgw_client` when this file is named `services.cluster`?

BTW, Pylint should fail here, since this `logger` variable was added, but never used. If not failing, our current config is broken and we should fix it.

Reply:
Removed